code/__DATASTRUCTURES/linked_lists.dm

```diff
@@ -0,0 +1,173 @@
+
+//Ok so it's technically a double linked list, bite me.
+
+/datum/linked_list
+	var/list/contents = list()
```

Comment:
@Miauw62 R/Bs+Dynamic(expanding) Arrays, R/Bs are only used for associations though, p.sure.